docs/faq.md

```diff
@@ -533,12 +533,13 @@ com.google.api.client.http.level=CONFIG
 
 And then launch your build tool as follows:
 ```sh
-mvn -Djava.util.logging.config.file=path/to/log.properties -DjibSerialize=true ...
+mvn -Djava.util.logging.config.file=path/to/log.properties -DjibSerialize=true -Djib.console=plain ...
 ```
 or
 ```sh
-gradle -Djava.util.logging.config.file=path/to/log.properties -DjibSerialize=true ...
+gradle -Djava.util.logging.config.file=path/to/log.properties -DjibSerialize=true -Djib.console=plain ...
 ```
+Note there is no dot(.) in `jibSerialize`.
```

Comment:
I feel like I've brought this up before and can't remember where, but I wonder if we should file an issue about making these system property names more consistent. Maybe for 2.0?

Reply:
yeah we can make the change now and deprecate jibSerialize (marked for removal in 2.0)

Reply:
https://github.com/GoogleContainerTools/jib/issues/1967